dnf/sack.py

```diff
@@ -57,6 +57,7 @@ def _build_sack(base):
 
 
 def _rpmdb_sack(base):
+    # used by subscription-manager (src/dnf-plugins/product-id.py)
```

Comment:
I'm not sure whether create rpmdb_sack(base) API function or ask subman to implement this on their side.

Reply:
Probably turn it into a public API, but that should be a new PR.